src/index.d.ts

```diff
@@ -38,7 +39,7 @@ export interface ArrowContainerProps extends UseArrowContainerProps {
 }
 
 export interface UsePopoverProps {
-  childRef: React.MutableRefObject<HTMLElement>;
+  childRef: React.MutableRefObject<HTMLElement | undefined>;
```

Comment:
In `Popover.tsx` - childRef has the type `React.MutableRefObject<HTMLElement | undefined>` and so the same type is required here as `childRef.current` can be `undefined`